<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1072</width>
    <height>742</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Foldabilizer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources/Resource.qrc">
    <normaloff>:/Resources/icon.png</normaloff>:/Resources/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="screens">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="DesignScreen">
       <attribute name="title">
        <string>Design</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QFrame" name="DesignScreenFrame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <property name="margin">
            <number>9</number>
           </property>
           <property name="spacing">
            <number>6</number>
           </property>
           <item row="0" column="0" rowspan="2" colspan="2">
            <layout class="QVBoxLayout" name="designLayout">
             <item>
              <widget class="QFrame" name="designFrame">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>1008</width>
                 <height>605</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>Main Toolbar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNewScene"/>
   <addaction name="separator"/>
   <addaction name="actionImportObject"/>
   <addaction name="separator"/>
   <addaction name="actionExportObject"/>
  </widget>
  <action name="actionNewScene">
   <property name="icon">
    <iconset resource="Resources/Resource.qrc">
     <normaloff>:/Resources/toolIcons/newScene.png</normaloff>:/Resources/toolIcons/newScene.png</iconset>
   </property>
   <property name="text">
    <string>New Scene</string>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionImportObject">
   <property name="icon">
    <iconset resource="Resources/Resource.qrc">
     <normaloff>:/Resources/toolIcons/insertObject.png</normaloff>:/Resources/toolIcons/insertObject.png</iconset>
   </property>
   <property name="text">
    <string>Import Object</string>
   </property>
   <property name="toolTip">
    <string>Import Object</string>
   </property>
  </action>
  <action name="actionExportObject">
   <property name="icon">
    <iconset resource="Resources/Resource.qrc">
     <normaloff>:/Resources/toolIcons/export.png</normaloff>:/Resources/toolIcons/export.png</iconset>
   </property>
   <property name="text">
    <string>Export Object</string>
   </property>
   <property name="toolTip">
    <string>Export Object</string>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Resources/Resource.qrc"/>
 </resources>
 <connections/>
</ui>
